root/static/scripts/common/constants/editTypes.js

```diff
@@ -25,3 +25,4 @@ export const EDIT_RELATIONSHIP_CREATE: 90 = 90;
 export const EDIT_RELATIONSHIP_EDIT: 91 = 91;
 export const EDIT_RELATIONSHIP_DELETE: 92 = 92;
 export const EDIT_RELEASE_REORDER_MEDIUMS: 13 = 13;
+export const EDIT_SERIES_EDIT: 141 = 141;
```

Comment:
For readability, it would be nice to insert it just before `EDIT_RELEASE_REORDER_MEDIUMS`, see previous comment.